ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/ServerUtils.java

```diff
@@ -97,6 +97,20 @@ public static String convertCommaSeparatedWilcardsToRegex(final String csv) {
     return out.toString();
   }
 
+  // See ParserUtil#getIdentifierText()
+  public static String getIdentifierText(final String text) {
```

Comment:
It's not great that this logic is repeated here and also in the method of the same name in ParserUtil.java. I couldn't think of a way to combine them, though, since the ParserUtil method is specific to the AST.